[2024-11-20T20:25:13.139+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-20T20:25:13.163+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_to_postgres manual__2024-11-20T20:24:52.157172+00:00 [queued]>
[2024-11-20T20:25:13.179+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_to_postgres manual__2024-11-20T20:24:52.157172+00:00 [queued]>
[2024-11-20T20:25:13.180+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-20T20:25:13.200+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2024-11-20 20:24:52.157172+00:00
[2024-11-20T20:25:13.210+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1296) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-20T20:25:13.212+0000] {standard_task_runner.py:72} INFO - Started process 1298 to run task
[2024-11-20T20:25:13.212+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_to_postgres', 'manual__2024-11-20T20:24:52.157172+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpcm_a9q40']
[2024-11-20T20:25:13.213+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask load_to_postgres
[2024-11-20T20:25:13.270+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_to_postgres manual__2024-11-20T20:24:52.157172+00:00 [running]> on host 9d6dc26aeebb
[2024-11-20T20:25:13.623+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-11-20T20:24:52.157172+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-20T20:24:52.157172+00:00'
[2024-11-20T20:25:13.624+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-20T20:25:13.625+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-20T20:25:13.626+0000] {logging_mixin.py:190} INFO - Current task name:load_to_postgres state:running start_date:2024-11-20 20:25:13.165087+00:00
[2024-11-20T20:25:13.627+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline and current dag run status:running
[2024-11-20T20:25:13.627+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-20T20:25:13.658+0000] {base.py:84} INFO - Retrieving connection 'skincare_connection'
[2024-11-20T20:25:13.838+0000] {load.py:46} INFO - Successfully inserted 674 records into the database.
[2024-11-20T20:25:13.839+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-20T20:25:13.849+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-20T20:25:13.850+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_to_postgres, run_id=manual__2024-11-20T20:24:52.157172+00:00, execution_date=20241120T202452, start_date=20241120T202513, end_date=20241120T202513
[2024-11-20T20:25:13.864+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-20T20:25:13.865+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-20T20:25:13.867+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline queued_at:2024-11-20 20:24:52.452751+00:00
[2024-11-20T20:25:13.868+0000] {logging_mixin.py:190} INFO - Task hostname:9d6dc26aeebb operator:PythonOperator
[2024-11-20T20:25:13.911+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-20T20:25:13.941+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-20T20:25:13.947+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
